app/views/repositories/revision.html.erb

```diff
@@ -27,28 +27,29 @@ See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
 
-<% content_for :action_menu_specific do %>
-  &#171;
-  <% unless @changeset.previous.nil? -%>
-    <%= link_to_revision(@changeset.previous, @project, :text => l(:label_previous)) %>
-  <% else -%>
-    <%= l(:label_previous) %>
-  <% end -%>
-  |
-  <% unless @changeset.next.nil? -%>
-    <%= link_to_revision(@changeset.next, @project, :text => l(:label_next)) %>
-  <% else -%>
-    <%= l(:label_next) %>
-  <% end -%>
-  &#187;&nbsp;
-
-  <%= form_tag({:controller => '/repositories', :action => 'revision', :project_id => @project}, :method => :get) do %>
-    <%= text_field_tag 'rev', @rev, :size => 8 %>
-    <%= submit_tag 'OK', :name => nil, class: 'button -highlight' %>
-  <% end %>
+<% content_for :toolbar do %>
+  <li class="toolbar-item">
+    <% if @changeset.previous.nil? %>
+      <button class="button" disabled="disabled"><%= l(:label_previous) %></button>
+    <% else %>
+      <%= link_to_revision(@changeset.previous, @project, text: l(:label_previous), class: 'button') %>
```

Comment:
This might be picky, but I find it irritating that in the enabled case we use an `a` and in the disabled case we use a `button`. Not sure if there are clients that would make a difference, though.

Is there a technical reason to do this that I don't see?